tests/subscription_handlers/test_handlers.py

```
@@ -0,0 +1,365 @@
+
+@pytest.fixture
+def simple_schema():
+    type_defs = """
+        type Query {
+            hello: String!
+        }
+    """
+    return make_executable_schema(type_defs)
+
+
+def test_http_handler_with_subscription_handlers(simple_schema):
```

Ruff (0.15.1)

[warning] Unused function argument: `simple_schema`

(ARG001)